### PR TITLE
Switch to new Travis configuration format

### PR DESCRIPTION
**Pull request by st-pasha**
Travis now has native R support, changing the .yaml file to use the new config format.
See https://github.com/craigcitro/r-travis/wiki/Porting-to-native-R-support-in-Travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,38 +1,24 @@
...
-  - mv tmp.sh travis-tool.sh
-  - chmod 755 ./travis-tool.sh  
-  - ./travis-tool.sh bootstrap
-install:
-  - ./travis-tool.sh bioc_install IRanges
-  - ./travis-tool.sh bioc_install GenomicRanges
-  - ./travis-tool.sh install_deps
-  
-script:
-  - echo "Commit:" $TRAVIS_COMMIT >> ./DESCRIPTION
```
Comment (jangorecki) [inline .travis.yml]: This info got lost.

File: .travis.yml [hunk 2]
```diff
@@ -1,38 +1,24 @@
...
-  - ./travis-tool.sh run_tests
+bioc_packages:
+  - IRanges
+  - GenomicRanges
 
-after_failure:
-  - ./travis-tool.sh dump_logs
+r_github_packages:
```
Comment (jangorecki) [inline .travis.yml]: We might not need to install them from github, CRAN versions were most likely updated till now.

File: .travis.yml (hunk 1 above)
Comment (mattdowle) [inline .travis.yml]: @jangorecki  Is it as simple as adding that line back in?  I'm looking in upgrade instructions and I can't see "script:" mentioned.

Comment (jangorecki) [inline .travis.yml]: Putting this into `before_install` is probably a way to go, at least this one is documented in travis native r doc. This metadata is used to check if current installed version commit matches to the one in drat. No need to install or download pkg then. If there is no such metadata it needs to download and reinstall every time.

**Pull request merged**